src/pages/[platform]/build-a-backend/auth/app-uninstall/index.mdx

```diff
@@ -0,0 +1,40 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Uninstalling the app',
+  description:
+    'Understand how to handle persistent data on a device when a user uninstalls the app.',
+  platforms: ['swift', 'android']
```

Comment:
Doesn't this also apply to Flutter?

Reply:
Nope with the latest Flutter library version the platform dependencies from the underlying libraries should be gone. Current docs only show it for native as wlel.